src/api/routes/signup.routes.js

```diff
@@ -0,0 +1,70 @@
+import express from 'express';
+import { signUpDriver, signUpOperator } from '../controllers/signup.controller.js';
+
+const router = express.Router();
+
+/**
+ * @swagger
+ * /signup/driver:
+ *   post:
+ *     summary: Create a new driver account
+ *     tags: [Driver]
+ *     security:
+ *       - bearerAuth: []
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             properties:
+ *               email:
+ *                 type: string
+ *               name:
+ *                 type: string
+ *             required:
+ *               - email
+ *               - name
+ *     responses:
+ *       200:
+ *         description: Successfully created driver account
+ *       400:
+ *         description: Invalid request body
+ *       401:
+ *         description: Unauthorized
+ */
+router.post('/driver', signUpDriver);
+
+/**
+ * @swagger
+ * /signup/operator:
+ *   post:
+ *     summary: Create a new operator account
+ *     tags: [Operator]
+ *     security:
+ *       - bearerAuth: []
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             properties:
+ *               email:
+ *                 type: string
+ *               name:
+ *                 type: string
+ *             required:
+ *               - email
+ *               - name
+ *     responses:
+ *       200:
+ *         description: Successfully created operator account
+ *       400:
```

Comment:
- I want only one endpoint
- add middleware to check if a user is not already registered by email
- our request body will be like this:
![Screenshot from 2023-03-28 11-46-51](https://user-images.githubusercontent.com/42033331/228197773-64a9824d-ecc0-4e55-84fb-692cd650b574.png)